cli.js

```diff
@@ -55,6 +55,12 @@ const cli = meow(`
 
 (async () => {
 	try {
+		const {rename} = cli.flags;
+		const renameReg = /{{(\s*)?basename(\s*)?}}/;
```

Comment:
Don't use abbreviations for the variable name.

Reply:
@sindresorhus It has been modified.